src/tellor_disputables/data.py

```diff
@@ -422,9 +427,24 @@ async def parse_new_report_event(
     monitored_feed = None
 
     for mf in monitored_feeds:
-
-        if get_query_type(mf.feed.query) == new_report.query_type:
-
+        try:
+            feed_qid = HexBytes(mf.feed.query.query_id).hex()
+        except Exception as e:
+            logger.error(f"Error while assembling query id: {mf.feed.query.descriptor}, {e}")
+            feed_qid = None
+
+        if feed_qid is None:
+            if get_query_type(mf.feed.query) == new_report.query_type:
+                # for generic queries the query params are None
+                if are_all_attributes_none(mf.feed.query):
+                    source = get_source_from_data(event_data.args._queryData)
+                    if source is None:
```

Comment:
Should this condition occurring be tested. Also, the added `are_all_attributes_none` case?